astropy/modeling/bounding_box.py

```diff
@@ -494,7 +494,7 @@ def _evaluate(
     ):
         """
         Perform model evaluation steps:
-            prepare_inputs -> evaluate -> prepare_outputs
+            prepare_inputs -> evaluate -> prepare_outputs.
```

Comment:
These and the following ones do not seem very good. Should it be a `::` in the line above?

Reply:
I compressed this to a summary line.